### PR TITLE
Toggle useDocumentTypes and useNativeBooleans defaults to true

### PR DESCRIPTION
**Pull request by fishcharlie**
I'm pretty sure this is a **🚨 breaking change 🚨**, but haven't run enough tests to verify yet.

This PR sets the useDocumentTypes and useNativeBooleans default options to `true` instead of `false`. This is the final major PR of v1.0.

### CODE REVIEW

**Comment by fishcharlie** [comment]
I **THINK** I recently ran into this problem just by using useDocumentTypes and useNativeBooleans on the current release version. This makes me wonder if it's just a bug in the code that we need to track down and find.

**Comment by brandongoode** [comment]
Looking at the changes now.  Sorry for the delay.

File: test/Schema.js [hunk 1]
```diff
@@ -154,9 +154,9 @@ describe('Schema tests', function (){
     schema.attributes.aObject.type.name.should.eql('object');
     should.exist(schema.attributes.aObject.default);
```
Comment (brandongoode) [inline test/Schema.js]: Instead of changing these tests, it may be more useful to set useDocumentTypes and useNativeBooleans to false.

**Comment by fishcharlie** [comment]
@brandongoode Just addressed those comments. Looks like those tests I mentioned earlier are still failing. You able to take a look at those? I can't figure out why they are failing.

**Comment by brandongoode** [comment]
@fishcharlie I'm pull down the PR and see if I can figure out what's failing.

**Comment by fishcharlie** [comment]
@brandongoode Thanks so much!!

**Comment by fishcharlie** [comment]
@brandongoode Does the branch `fishcharlie-dynamodbtypesdefaulttoggle` fix this? Or is it still in progress? Looks like that branch is passing the CI tests. Not sure if you are still working on it tho.

**Comment by brandongoode** [comment]
@fishcharlie Yes.  I've modified a few of the test to resolve the errors you were having.  I also removed the logic to convert S types to Objects since with `useDocumentTypes: true` S should always be a String.

**Pull request merged**